corehq/apps/reports/views.py

```diff
@@ -1415,7 +1415,12 @@ def _get_form_render_context(request, domain, instance, case_id=None):
     # Build ordered list of questions and dict of question values => responses
     # Question values will be formatted to be processed by XFormQuestionValueIterator,
     # for example "/data/group/repeat_group[2]/question_id"
-    question_response_map, ordered_question_values = get_data_cleaning_data(form_data, instance)
+    try:
+        question_response_map, ordered_question_values = get_data_cleaning_data(form_data, instance)
+    except AttributeError as err:
+        question_response_map, ordered_question_values = (None, None)
```

Comment:
How does the rest of the page rendering and javascript handle `None`/`null` values here? Can you confirm there aren't javascript errors on the page after this?

Reply:
There are no JS errors that are coming up in the console. 
And I cannot see any difference after raising AttributeError in the page that is rendered.
I am not sure of the data cleaning button, I can't find it on the page in both the cases
Here is the screenshot when the Exception is raised-
<img width="1790" alt="Screenshot 2020-04-14 at 9 03 44 PM" src="https://user-images.githubusercontent.com/7694243/79243906-deb64180-7e93-11ea-9864-095b080fcff2.png">

Reply:
Ah, okay that button apparently only shows up when a user / project has access to the feature. I think the easy way to test it is to (temporarily locally) change this line https://github.com/dimagi/commcare-hq/blob/ad11f395452982934f1d8f5151f9d27f9bc14358/corehq/apps/reports/views.py#L1628
to
```
        "show_edit_submission": True,
```

Reply:
When exception is raised and you click on `Clear Form Submission`, the UI shows `No Editable Properties found`

<img width="1128" alt="Raise-Attr-Err" src="https://user-images.githubusercontent.com/7694243/79306500-f254bd00-7f12-11ea-9cda-43009816549d.png">

And if I go ahead and click on `Save` it gives the following error on UI.

<img width="601" alt="Errored_SS" src="https://user-images.githubusercontent.com/7694243/79306669-48296500-7f13-11ea-95b8-a41e3815e014.png">

There were no JS errors in the console.